lib/fuzzy-finder-view.js

```diff
@@ -299,21 +358,49 @@ class FuzzyFinderView {
   projectRelativePathsForFilePaths (filePaths) {
     // Don't regenerate project relative paths unless the file paths have changed
     if (filePaths !== this.filePaths) {
-      const projectHasMultipleDirectories = atom.project.getDirectories().length > 1
       this.filePaths = filePaths
-      this.projectRelativePaths = this.filePaths.map((filePath) => {
-        const [rootPath, projectRelativePath] = atom.project.relativizePath(filePath)
-        const label =
-          rootPath && projectHasMultipleDirectories
-            ? path.join(path.basename(rootPath), projectRelativePath)
-            : projectRelativePath
-
-        return {uri: filePath, filePath, label}
-      })
+      this.projectRelativePaths = this.filePaths.map(
+        (filePath) => this.convertPathToSelectViewObject(filePath)
+      )
     }
 
     return this.projectRelativePaths
   }
+
+  convertPathToSelectViewObject (filePath) {
+    const projectHasMultipleDirectories = atom.project.getDirectories().length > 1
+
+    const [rootPath, projectRelativePath] = atom.project.relativizePath(filePath)
+    const label =
+      rootPath && projectHasMultipleDirectories
+        ? path.join(path.basename(rootPath), projectRelativePath)
+        : projectRelativePath
+
+    return {uri: filePath, filePath, label}
+  }
+
+  getAbsolutePath (relativePath) {
```

Comment:
This method is not great because it'll do a few fs accesses via `fs.existSync`, but it should not be very expensive because it does it against the list of filtered items (which is maximum `10` items).

This means that it'll do `10 * number of opened projects` file system accesses. Since I don't expect users having many projects opened in the same window, this should be fine